templates/demo.html

```diff
@@ -7,31 +7,36 @@
 {% block content %}
 <script type="module">
   import AnboxStream from '/static/js/anbox-stream-sdk.js';
-
-  let stream = new AnboxStream({
-  targetElement: "anbox-stream",
-  url: '{{ ANBOXCLOUD_API_BASE }}',
-  authToken: '{{ session["authentication_token"] }}',
-  session: {
-      app: "com.vectorunit.cobalt.canonical", // Given
-  },
-  screen: {
-    // Currently available max resolution
-    // To change to portrait just flip the values between width and height
-      width: 1280,
-      height: 720,
-  },
-  // Using keyboard controls to test it on non-touch device
-  controls: {
-    keyboard: true
-  },
-  callbacks: {
-    error: error => {
-      console.log("AnboxStream failed: " + error.message);
+  // @ params buttonId: HTML id of the button that activates it
+  // @ params anboxStreamId: HTML id attribute of the div containing the anbox stream
+  function streamContainer(anboxStreamId) {
+    let stream = new AnboxStream({
+    targetElement: anboxStreamId,
+    url: '{{ ANBOXCLOUD_API_BASE }}',
+    authToken: '{{ session["authentication_token"] }}',
+    session: {
+        app: "com.vectorunit.cobalt.canonical", // Given
+    },
+    screen: {
+      // Currently available max resolution
+      // To change to portrait just flip the values between width and height
+        width: 1280,
+        height: 720,
+    },
+    // Using keyboard controls to test it on non-touch device
+    controls: {
+      keyboard: true
+    },
+    callbacks: {
+      error: error => {
+        console.log("AnboxStream failed: " + error.message);
+      }
     }
+  });
+  stream.connect();
   }
-});
-stream.connect();
+
+  document.querySelector("#buggy-racing-btn").addEventListener('click', streamContainer("buggy-racing"))
```

Comment:
We might want to change it to `document.getElementById` ?

Reply:
Yeah, I'm going to slightly restructure the JS here, I think